packages/core-utils/src/time.js

```diff
@@ -75,10 +117,7 @@ export function formatTime(ms, options) {
  * @return {string}                   formatted text representation
  */
 export function formatSecondsAfterMidnight(seconds, timeFormat) {
-  return moment()
-    .startOf("day")
-    .seconds(seconds)
-    .format(timeFormat);
+  return format(add(startOfDay(new Date()), { seconds }), timeFormat);
```

Comment:
Non-blocking nit: This is a touch more difficult to read than the moment dot chaining. I sometimes prefer an interim variable here or there to help with legibility.
```suggestion
  const time = add(startOfDay(new Date()), { seconds })
  return format(time, timeFormat);
```

Reply:
Much cleaner thanks! Changed in 6db148be6701f51c7a26b67765a8b15409bb63de